### DISCUSSION TITLE
NewsNation Live News

### DISCUSSION THREAD

**Discussion by Central302** [iptv-org/community#753] [Q&A]
### Channel Name

NewsNation Live News

### Channel ID

NewsNationEast.us

### Country

USA-Nation Wide

### Language

English

### Website

https://www.newsnationnow.com/news-nation-live/ (Free Access Availble Only 5 Minitues)

### Notes

Cable Access Required, But Using Lura Live. https://tkx.mp.lura.live/rest/v2/mcp/video/adstXlnjeoByg4YE?anvack=GNLYj4WPEPgKWUvG6mTVNFejokMyVq3x 
Only News Broadcast.

**Comment by BellezaEmporium** [comment]
You got the Lura (Anvato) link. Basically the code I have for this is a regex that reads the link out of it.

**Comment by MapGuy11** [comment]
> You got the Lura (Anvato) link. Basically the code I have for this is a regex that reads the link out of it.

Works with your stuff btw I can send the code if you want me to. Bypasses 5min rule.

**Comment by BellezaEmporium** [comment]
> > You got the Lura (Anvato) link. Basically the code I have for this is a regex that reads the link out of it.
> 
> Works with your stuff btw I can send the code if you want me to. Bypasses 5min rule.

You can send the code if you want. I know how it looks like, have already dealt with Anvato-protected links before :)

**Comment by MapGuy11** [comment]
> > > You got the Lura (Anvato) link. Basically the code I have for this is a regex that reads the link out of it.
> > 
> > 
> > Works with your stuff btw I can send the code if you want me to. Bypasses 5min rule.
> 
> You can send the code if you want. I know how it looks like, have already dealt with Anvato-protected links before :)

Did you remove your repo as it 404's now?

**Comment by BellezaEmporium** [comment]
> > > > You got the Lura (Anvato) link. Basically the code I have for this is a regex that reads the link out of it.
> > > 
> > > 
> > > Works with your stuff btw I can send the code if you want me to. Bypasses 5min rule.
> > 
> > You can send the code if you want. I know how it looks like, have already dealt with Anvato-protected links before :)
> 
> Did you remove your repo as it 404's now?

Yes.

**Comment by iamacarpet** [comment]
@MapGuy11 @BellezaEmporium any chance you’d be able to share your script please?

I was sad to see News Nation US isn’t included in the channel list, as I’m trying to help people view the David Grusch interview tonight.

Any help greatly appreciated

**Comment by BellezaEmporium** [comment]
> @MapGuy11 @BellezaEmporium any chance you’d be able to share your script please?
> 
> I was sad to see News Nation US isn’t included in the channel list, as I’m trying to help people view the David Grusch interview tonight.
> 
> Any help greatly appreciated

```
fetch("https://tkx.mp.lura.live/rest/v2/mcp/video/adstXlnjeoByg4YE?anvack=GNLYj4WPEPgKWUvG6mTVNFejokMyVq3x")
	.then(response => response.text())
	.then(text => {
		var goodone = text.replace(/\\/g, "");
		var regex = new RegExp(/"embed_url":"(.+?)"/)
		var results = regex.exec(goodone)
        res.setHeader('Content-Type', 'application/vnd.apple.mpegurl')
        res.redirect(results[1])
   })
```

**Comment by iamacarpet** [comment]
Thanks @BellezaEmporium ,

I tried something very similar and it still seemed to be subject to the 5 minute limit.

Was there additional code from @MapGuy11 for that part ?

**Comment by BellezaEmporium** [comment]
Not AFAIK, since I had shared my implementation with him.

However, the current state of the stream seems to be not working, as it gets the same content on each refresh.

**Comment by iamacarpet** [comment]
Balls, yes, I got that after a while as well.

Just the same two adverts and no actual show?

I’m with family at the moment so can’t check, but I plan to spend an hour or two later seeing if I can get anything working.

I was hoping the channel was just off air due to the time difference, but it’s late enough in the US now that it should be broadcasting now, so it must be actually broken, which sucks!

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/us_moveonjoy.m3u#L52-L53
